packages/background/src/index.integration.test.js

```diff
@@ -156,6 +157,40 @@ describe('integration test', function () {
         expect(stat.result.networkId).toBe('0xbb7')
       })
     })
+
+    describe('wallet_accounts', function () {
+      test('wallet_accounts', async () => {
+        await request({
+          method: 'wallet_importMnemonic',
+          params: {mnemonic: MNEMONIC, password},
+        })
+        const accounts = await request({method: 'wallet_accounts'})
+        expect(accounts.result[0]).toBe(CFX_ACCOUNTS[0].base32)
+        await request({
+          method: 'wallet_addNetwork',
+          params: {
+            chainId: '0x539',
+            chainName: 'ethfoo',
+            nativeCurrency: {
+              name: 'ETH',
+              symbol: 'ETH',
+              decimals: DEFAULT_CURRENCY_DECIMALS,
+            },
+            rpcUrls: [ETH_LOCALNET_RPC_ENDPOINT + '/'],
+          },
+          _rpcStack: ['frombg'],
+          _internal: true,
+        })
+        const accountsAnother = await request({
+          method: 'wallet_accounts',
```

Comment:
should also test this from dapp env, with `_inpage: true`

Reply:
have already added the test case when the `_inpage` is `true`